tests/run_notebook.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+import nbformat
+from nbconvert.preprocessors import ExecutePreprocessor
+from nbformat import read
+
+
+def execute_notebook(notebook_path) -> None:
+    notebook = read(notebook_path, as_version=4)
+    execute_preprocessor = ExecutePreprocessor(timeout=600)
+
+    execute_preprocessor.preprocess(notebook, {"metadata": {"path": "."}})
+
+
+def add_parameters_to_cell(cell, take_screenshot):
+    # TODO: Change hardcoded values to variables
+    parameters = {
+        "coordinate_system_name": '"global"',
+        "_tested_notebook": '"test_notebook.ipynb"',
+        "_test_target": '"cell5"',
```

Comment:
Also the value of this variable is already present in the notebook, the script should not modify it.